daemon/cleanup.go

```diff
@@ -19,10 +19,10 @@ import (
 	"os"
 	"os/signal"
 	"sync"
-	"syscall"
 
 	"github.com/cilium/cilium/pkg/lock"
 	"github.com/cilium/cilium/pkg/pidfile"
+	"golang.org/x/sys/unix"
```

Comment:
Add an empty line before this import to separate cilium imports from non-cilium imports.